meeteval/wer/api.py

```diff
@@ -0,0 +1,181 @@
+import glob
+import logging
+import re
+from pathlib import Path
+
+import meeteval.io
+from meeteval.wer.wer import ErrorRate
+
+__all__ = [
+    'cpwer',
+    'orcwer',
+    'mimower',
+    'tcpwer',
+]
+
+
+def _glob(pathname):
+    # Normalize and glob (for backwards compatibility) the path input
+    match = list(glob.glob(pathname))
+    # Forward pathname if not matched to get the correct error message
+    return match or [pathname]
+
+
+def _maybe_load(paths, file_format) -> meeteval.io.SegLST:
+    if isinstance(paths, (str, Path)):
+        paths = [paths]
+    if isinstance(paths, (tuple, list)):
+        paths = [Path(file) for r in paths for file in _glob(str(r))]
+        return meeteval.io.asseglst(
+            meeteval.io.load(paths, format=file_format))
+    else:
+        try:
+            return meeteval.io.asseglst(paths)
+        except Exception as e:
+            raise TypeError(type(paths), paths) from e
+
+
+def _load_texts(
```

Comment:
Are there any operations that might span across reference and hypothesis? Currently there's not, and then we could simplify to the signature `_load_text(paths, regex, sort, file_format)` and call it one time for the reference and one time for the hypothesis

Reply:
Is `_load_text` even a good name anymore? It now supports much more than text, e.g., RTTM. `_load` could be enough, although it clashes with `_maybe_load`